packages/remix-css-bundle/README.md

```diff
@@ -0,0 +1,9 @@
+# @remix-run/css-bundle
```

Comment:
Could we maybe call this `@remix-run/css`?

![751mnd](https://user-images.githubusercontent.com/3082153/208848061-2d58cc31-62e5-4964-8151-6440213634d7.jpg)

Reply:
Nah, css-bundle is way more descriptive.

Reply:
Definitely looks cleaner, but I agree with @mjackson, I think it's a bit too vague.